{{cookiecutter.project_slug}}/.github/workflows/release.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       uses: pypa/gh-action-pypi-publish@release/v1
 
     - name: sign
-      uses: sigstore/gh-action-sigstore-python@v2.1.1
+      uses: sigstore/gh-action-sigstore-python@v3.0.0
       with:
         inputs: ./dist/*.tar.gz ./dist/*.whl
         release-signing-artifacts: true
```

Comment:
IMO let's remove this step entirely and enable attestation generation on the publishing step above 🙂

Reply:
Fixed!